api/cmd/sftype/test_sftype.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Usage: find . | entr -c docker exec e-qip-prototype_api_1 ./cmd/sftype/test_sftype.sh
```

Comment:
Cool – this is a good first step towards something more robust that we can copy for some of our other tools.